testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -29,25 +29,37 @@
 When(/^I stop salt-minion on "(.*?)"$/) do |minion|
   node = get_target(minion)
   ## Uyuni uses Salt Bundle. Package name is "venv-salt-minion"
-  pkgname = $product == 'Uyuni' ? "venv-salt-minion" : "salt-minion"
-  node.run("rc#{pkgname} stop", check_errors: false) if minion == 'sle_minion'
-  node.run("systemctl stop #{pkgname}", check_errors: false) if %w[ceos_minion ubuntu_minion kvm_server xen_server].include?(minion)
+  pkgname = $product == 'Uyuni' ? 'venv-salt-minion' : 'salt-minion'
+  os_version, os_family = get_os_version(node)
+  if os_family =~ /^sles/ && os_version =~ /^11/
+    node.run("rc#{pkgname} stop", check_errors: false)
+  else
+    node.run("systemctl stop #{pkgname}", check_errors: false)
+  end
 end
 
 When(/^I start salt-minion on "(.*?)"$/) do |minion|
   node = get_target(minion)
   ## Uyuni uses Salt Bundle. Package name is "venv-salt-minion"
-  pkgname = $product == 'Uyuni' ? "venv-salt-minion" : "salt-minion"
-  node.run("rc#{pkgname} restart", check_errors: false) if minion == 'sle_minion'
-  node.run("systemctl restart #{pkgname}", check_errors: false) if %w[ceos_minion ubuntu_minion kvm_server xen_server].include?(minion)
+  pkgname = $product == 'Uyuni' ? 'venv-salt-minion' : 'salt-minion'
+  os_version, os_family = get_os_version(node)
+  if os_family =~ /^sles/ && os_version =~ /^11/
```

Comment:
Has this considered other systems included only in build validation setup?

Reply:
@Bischoff CentOS6 is not anymore in our BV envs, neither in CI. We are good about that one.
But I'm not sure about other cases, like debians, centos-like, others